src/hyperlight_host/src/sandbox/mod.rs

```diff
@@ -177,29 +171,19 @@ mod tests {
     #[test]
     // TODO: add support for testing on WHP
     #[cfg(target_os = "linux")]
-    fn test_is_hypervisor_present() {
+    fn is_hypervisor_present() {
         // TODO: Handle requiring a stable API
```

Comment:
This comment can go since mshv got rid of the stable api flag

Reply:
fixed